Example/Example/ViewControllers/DemoViewController.swift

```diff
@@ -111,6 +146,22 @@ extension DemoViewController {
         present(controller, animated: true, completion: nil)
     }
 
+    fileprivate func selectOperator(operators: [Operator], completion: @escaping (Operator) -> Void) {
```

Comment:
Aren't we already selecting operators here https://github.com/salemove/ios-bundle/pull/7/files#diff-33044241b20ee33162a4d728f6328f3fL196?

Reply:
Nvm, explained in the commit message.
Should https://github.com/salemove/ios-bundle/pull/7/files#diff-33044241b20ee33162a4d728f6328f3fL196 be removed now?